Code/Mantid/MantidPlot/src/ConfigDialog.cpp

```diff
@@ -2465,10 +2492,14 @@ void ConfigDialog::updateMdPlottingSettings()
     m_mdSettings.setUserSettingColorMap(vsiDefaultColorMap->currentText());
   }
 
-  // Read if the usage of the last color map should be performed
-  if (vsiLastSession)
+  // Read if the usage of the last color map and background color should be performed
+  if (mdPlottingVsiFrameBottom->isChecked())
+  {
```

Comment:
You could have compressed 8 lines down to 1 line with m_mdSettings.setUsageLastSession(!mdPlottingVsiFrameBottom->isChecked());